Examples/Ising1d/ising1d.py

```diff
@@ -34,13 +34,13 @@
 op = nk.optimizer.Sgd(learning_rate=0.1)
```

Comment:
sorry, very last thing, can we revert this example?

Reply:
No problem. Should it use the old C++ driver or the new one correctly?

Reply:
The new one I'd say... then at some point we should deprecate the old style things, but we need to support also old-style variational at least until the major release

Reply:
Agreed.

Reply:
Hm, I'd leave the example as it is (beside making it run again, of course) right now, since the driver does not have a `run` function like the old driver yet.
(Personally, I would not miss that function if stays absent from the new driver. But if necessary, we could still add it later.)